deploy/prometheus/prometheus-alertmanager-template.yaml

```diff
@@ -7,37 +7,71 @@ metadata:
 type: Opaque
 stringData:
   notifications.tmpl: |-
-    {{ define "__single_message_title" }}{{ range .Alerts.Firing }}{{ .Labels.alertname }} @ {{ .Annotations.identifier }}{{ end }}{{ range .Alerts.Resolved }}{{ .Labels.alertname }} @ {{ .Annotations.identifier }}{{ end }}{{ end }}
-    {{ define "custom_title" }}[ {{ .Status | toUpper }}:{{ if eq .Status "firing" }} :fire: {{ .Alerts.Firing | len }}{{ else }} :ok_hand: {{ .Alerts.Resolved | len }}{{ end }} ] {{ if or (and (eq (len .Alerts.Firing) 1) (eq (len .Alerts.Resolved) 0)) (and (eq (len .Alerts.Firing) 0) (eq (len .Alerts.Resolved) 1)) }}{{ template "__single_message_title" . }}{{ end }}{{ end }}
-    {{ define "custom_slack_message" }}
-    {{ range .Alerts.Firing }}
-    *:fire: fired {{ .Labels.severity | toUpper }} {{ .Annotations.summary }}*
-    {{ .Annotations.description }}
-    {{ end }}
-    {{ range .Alerts.Resolved }}
-    *:ok: resolved {{ .Labels.severity | toUpper }} {{ .Annotations.summary }}*
-    {{ .Annotations.description }}
+    {{ define "__subject" }}
+      [{{ if eq .Status "firing" }}FIRE:{{len .Alerts.Firing}}{{ else }}OK{{ end }}] {{.GroupLabels.alertname}}
     {{ end }}
+
+    {{ define "slack_message" }}
+      {{ range .Alerts.Firing -}}
+       {{- range .Annotations.SortedPairs }}
+      *{{   .Name }}*: {{ .Value }}
+       {{- end }}
+
+       {{ range .Labels.SortedPairs -}}
+        {{- if and (ne .Name "alertname") (ne .Name "severity") (ne .Name "tenant") }}
+      {{     .Name}} = `{{.Value}}`
+        {{- end }}
+       {{- end }}
+    ---
+      {{ end }}
+
+      {{ range .Alerts.Resolved -}}
+       {{- range .Annotations.SortedPairs }}
+        {{- if (ne .Name "description") }}
```

Comment:
@varuzam could you look to these changes, I exclude description for `resolved` message
cause prometheus applied template to alert message only when alert `expr` results is not empty 
so, on resolved alert state description will get "old" results for `| query` template